aiidalab_widgets_base/viewers.py

```diff
@@ -167,15 +168,16 @@ class _StructureDataBaseViewer(ipw.VBox):
     selection = List(Int)
     selection_adv = Unicode()
     supercell = List(Int)
-    cell = Instance(Cell, allow_none=True)
+    cell = Instance(Cell, allow_none=True)  # Will be deprecated
+    ase_structure = Instance(Atoms, allow_none=True)
     DEFAULT_SELECTION_OPACITY = 0.2
     DEFAULT_SELECTION_RADIUS = 6
     DEFAULT_SELECTION_COLOR = "green"
 
     def __init__(
         self,
         configure_view=True,
-        configuration_tabs=["Selection", "Appearance", "Cell", "Download"],
+        configuration_tabs=["Cell", "Selection", "Appearance", "Download"],
```

Comment:
[One should not use mutable default values in Python.](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)